docs/crio.8.md

```diff
@@ -139,7 +139,7 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--decryption-keys-path**="": Path to load keys for image decryption. (default: "/etc/crio/keys/")
 
-**--default-capabilities**="": Capabilities to add to the containers (default: ["CHOWN" "DAC_OVERRIDE" "FSETID" "FOWNER" "NET_RAW" "SETGID" "SETUID" "SETPCAP" "NET_BIND_SERVICE" "SYS_CHROOT" "KILL"])
+**--default-capabilities**="": Capabilities to add to the containers (default: ["CHOWN" "DAC_OVERRIDE" "FSETID" "FOWNER" "SETGID" "SETUID" "SETPCAP" "NET_BIND_SERVICE" "KILL"]) (default: [])
```

Comment:
should we drop the second `(default: [])`?

Reply:
Yes, this can be fixed with https://github.com/cri-o/cri-o/pull/3119#discussion_r367373186